modules/ding_dibs/ding_dibs.admin.inc

```diff
@@ -62,5 +62,17 @@ function ding_dibs_admin_settings($form, &$form_state) {
     '#description' => t('DIBS URL as per http://tech.dibs.dk/10_step_guide/, ie.: https://payment.architrade.com/cgi-bin/capture.cgi'),
   );
 
+  $form['wipe_transaction_threshold'] = array(
```

Comment:
This varible should be rename to ding_dibs_transaction_wipe_threshold (prefixed with module name).